bower.json

```diff
@@ -12,15 +12,19 @@
     "paper-elements": "PolymerElements/paper-elements#^1.0.1",
     "platinum-elements": "PolymerElements/platinum-elements#^1.0.1",
     "svg-piechart": "~1.0.1",
-    "akc-meta": "git://github.com/akc42/akc-meta.git#b0a6bde0e",
     "file-upload": "~1.6.1",
     "material-progress": "^0.1.2",
-    "paper-tabs": "PolymerElements/paper-tabs#^1.6.2"
+    "paper-tabs": "PolymerElements/paper-tabs#^1.6.2",
+    "app-route": "polymerelements/app-route#^0.9.2",
+    "iron-lazy-pages": "timvdlippe/iron-lazy-pages#^1.4.1"
   },
   "devDependencies": {
     "web-component-tester": "*",
     "test-fixture": "PolymerElements/test-fixture#^1.0.0"
   },
   "appPath": "app",
-  "moduleName": "planningtoolApp"
+  "moduleName": "planningtoolApp",
+  "resolutions": {
+    "paper-input": "spinner-mixin"
```

Comment:
Yeah, but it was complaining on bower install. Can probably be removed after the other PR is merged and this PR is rebased.